src/service/video-manager-impl.js

```diff
@@ -894,13 +945,235 @@ class VideoEntry {
     // Update docking state
     if (this.scrollMap_(DOCK_SCALE, 1) == DOCK_SCALE) {
       this.dockState_ = DockStates.DOCKED;
+      this.initializeDragging_();
+      this.drag_();
```

Comment:
mmm, `drag_()` keeps running every rAF, not great. We need to only run it with `touch move`.